homeassistant/components/local_todo/config_flow.py

```diff
@@ -0,0 +1,40 @@
+"""Config flow for Local To-do integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import CONF_TODO_LIST_NAME, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_TODO_LIST_NAME): str,
+    }
+)
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Local To-do."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle the initial step."""
+        errors: dict[str, str] = {}
+        if user_input is not None:
+            return self.async_create_entry(
```

Comment:
Since the slug name will be used as a path, we should not allow creating more than one config entry with the same slug name.

Reply:
Added a check and a test for this. 

I am not sure if the unique id is the right way to do this, given most of our rules are focused on external devices or cloud APIs, but in this case since its a slug that can't be changed by the user it seems like it can meet the spirit of the requirements.

Let me know if you had another specific technical implementation in mind (the one i can think of is creating a separate field in the data with the slug name and checking that instead)

Reply:
I suggest just using `_async_abort_entries_match`.